### DISCUSSION TITLE
Test build available for 1.4.3.0

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#2499] [Announcements]
Hi! A non-pre-release test-build is available from the latest commit sha: https://github.com/Pryaxis/TShock/actions/runs/1483941680

If you know how to use this, please let us know how stable it is, before we push a pre-release out with it. Due to a lot of technical reasons involving me being on macOS, I don't have a proper development environment to test everything fully with right this moment. We obviously want to get a release out, though, so I'd love community help to test this build out.

**Comment by HiddenStriker** [comment]
Er, how am I supposed to run this? Since it's just the exe, I tried running it standalone, I tried putting it in my TShock folder (akin to updating) and I also tried renaming it into "TerrariaServer.exe" and putting the old server into a backup folder to run it, but all of this has gotten me:

Unhandled Exception: System.IO.FileNotFoundException: 'NDesk.Options, Version=0.2.1.0, Culture=neutral, PublicKeyToken=null' failed to load the file or assembly or one of its dependencies. The system cannot find the file specified.
Location: TShock.Modifications.Bootstrapper.Program.Main(String[] args)

Edit: My bad, windows was being a dork by forcefully updating my PC, which is why I assume my browser didn't allow me to scroll down and see the TShock at the time, and I downloaded the OTAPI bootstrapper. I will try the TShock file now.

**Comment by HiddenStriker** [comment]
+Seems to work okay for me. I tried out all the new items (id numbers 5088 and on from what I could find) and they all work fine. Even the heal command overhaul seems to made its way in :D. The only thing I couldn't get to work at first was the /spawnboss deerclops command but doing it with /spawnmob deerclops worked instead.

We tested this out with three people on my server, and the only "notable" thing we found was that the Weather Pain does not ignore 10 DEF in PvP like advertised, but that might be a Terraria thing more than a TShock thing. So I can attest that it works well enough for a small to very small scale use  (1-6 people) and haven't seen any instabilities in what little time I tested it in. Though note that we didn't try -every- command to see if they work, just the ones we use the most, such as warps, tp, heal, slap, kill, give item, say, boss spawns, time, worldevent control. And it was done on a "trusted friends" server where everyone is a superadmin so we can't give any insight on anti cheat measures, projectile rules, and all that sort of stuff.

(I used the non-debug TShock server, Win10 machine)

**Reply by hakusaro** [reply]
Thank you so much! I’ve created #2500 on your behalf.
